incubator/kafka/templates/statefulset.yaml

```diff
@@ -59,14 +85,12 @@ spec:
           initialDelaySeconds: 30
           timeoutSeconds: 5
         readinessProbe:
-          exec:
-            command:
-              - kafka-topics
-              - --zookeeper
-              - {{ template "zookeeper.url" . }}
-              - --list
-          initialDelaySeconds: 30
-          timeoutSeconds: 5
+          failureThreshold: 3
+          periodSeconds: 10
+          successThreshold: 1
+          tcpSocket:
```

Comment:
What was the point of this change again?

Reply:
Ahh okay, it's good that we pulled the change from the upstream that this chart is meant to be based on. It's more defensible if we try to merge these changes into the upstream chart.

Thanks